Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,14 +1,35 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+
+namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
         {
             // Print each of the statistical output using Text Table with padding 3:
             // - FindHighestBalanceEver
+
+            Console.WriteLine("The richest person ever was : ");
+            var answer = BalanceStats.FindHighestBalanceEver(PeoplesBalances.Balances);
+            Console.WriteLine(TextTable.Build(answer, 3));
+
             // - FindPersonWithBiggestLoss
+
+            //Console.WriteLine("The person with the biggest loss was  : ");
+            //answer = BalanceStats.FindPersonWithBiggestLoss(PeoplesBalances.Balances);
+            //Console.WriteLine(TextTable.Build(answer, 3));
+
             // - FindRichestPerson
+
+            Console.WriteLine("The richest person at this moment is : ");
```

Comment:
Function should be used instead of comments.

Reply:
I will try to figure this out.